receptor-python-worker/.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit = 
+    tests/*
```

Comment:
FYI, the best practice is to measure the coverage in test files. Moreover, it should be 100% at all times — otherwise, there'll be dead tests that never run.

https://nedbatchelder.com/blog/202008/you_should_include_your_tests_in_coverage.html